ftl/core/importing.ftl

```diff
@@ -79,3 +79,4 @@ importing-processed-media-file =
     }
 importing-importing-collection = Importing collection...
 importing-failed-to-import-media-file = Failed to import media file: { $debugInfo }
+importing-colpkg-too-new = Please update to the latest Anki version, then import the .colpkg file again.
```

Comment:
I frequently struggle with the placement of new strings. Should the string go where it's used in the source code (exporting in this case) or where it logically belongs from a user/translator perspective (importing)?

Reply:
Using the same prefix as the source filename makes it a bit easier to locate relevant strings - eg when editing exporting.rs one can type tr.exporting_ and see the strings that will be used there. But once modules are split into multiple files, or strings used in multiple locations, there's no one right answer :-) In terms of making things easier for translators, if we think it might not be clear, we can always add a comment explaining where/why the string is used.